### ISSUE TITLE
error: ‘else’ without a previous ‘if’

### ISSUE THREAD

**Issue by XiaoYu-HN** [hanslub42/rlwrap#112]
Hi expert, my OS is Oracle Linux 7.9, when run 'make install', it report error:
```
[root@oracle-19c-vagrant rlwrap]# make install
Making install in doc
make[1]: Entering directory `/home/oracle/rlwrap/doc'
sed -e 's#@DATADIR@#/usr/local/share#'  rlwrap.man > rlwrap.1
make[2]: Entering directory `/home/oracle/rlwrap/doc'
make[2]: Nothing to be done for `install-exec-am'.
 /bin/mkdir -p '/usr/local/share/man/man1'
 /bin/install -c -m 644 rlwrap.1 '/usr/local/share/man/man1'
make[2]: Leaving directory `/home/oracle/rlwrap/doc'
make[1]: Leaving directory `/home/oracle/rlwrap/doc'
Making install in src
make[1]: Entering directory `/home/oracle/rlwrap/src'
gcc -DHAVE_CONFIG_H -I. -I..    -DDATADIR=\"/usr/local/share\"  -g -O2 -MT main.o -MD -MP -MF .deps/main.Tpo -c -o main.o main.c
mv -f .deps/main.Tpo .deps/main.Po
gcc -DHAVE_CONFIG_H -I. -I..    -DDATADIR=\"/usr/local/share\"  -g -O2 -MT signals.o -MD -MP -MF .deps/signals.Tpo -c -o signals.o signals.c
signals.c: In function ‘mysignal’:
signals.c:71:3: error: ‘else’ without a previous ‘if’
   else if (handler == SIG_IGN)
   ^
signals.c:73:3: error: ‘else’ without a previous ‘if’
   else
   ^
make[1]: *** [signals.o] Error 1
make[1]: Leaving directory `/home/oracle/rlwrap/src'
make: *** [install-recursive] Error 1
```

**Comment by syrinxtech** [comment]
Getting same error as above.

```
kali@kali:~/tools/rlwrap/src$ make
gcc -DHAVE_CONFIG_H -I. -I..    -DDATADIR=\"/usr/local/share\"  -g -O2 -MT signals.o -MD -MP -MF .deps/signals.Tpo -c -o signals.o signals.c
signals.c: In function ‘mysignal’:
signals.c:71:2: error: ‘else’ without a previous ‘if’
   71 |  else if (handler == SIG_IGN)
      |  ^~~~
signals.c:73:3: error: ‘else’ without a previous ‘if’
   73 |   else
      |   ^~~~
make: *** [Makefile:381: signals.o] Error 1

```

**Comment by hanslub42** [comment]
This cpuld only heppen when configured without `--enable-debug`. Solved by 4bb15952618f2b1156ad75d508bafce2d32dd712.

**Comment by syrinxtech** [comment]
Thanks @hanslub42.

I recompiled with that switch and it took care of the issue.
